packages/react-native/ReactCommon/jsi/jsi/decorator.h

```diff
@@ -544,6 +547,10 @@ class WithRuntimeDecorator : public RuntimeDecorator<Plain, Base> {
     Around around{with_};
     return RD::evaluatePreparedJavaScript(js);
   }
+  void queueMicrotask(const Function& callback) override {
```

Comment:
https://github.com/facebook/hermes/commit/0a632b73b8386dfc9fde0ddca349919c20fe2852 adds this change in JSI. We need it in React Native as well